_redirects

```diff
@@ -0,0 +1 @@
+/docs/fanatical-support-aws/access-and-permissions/user-management-and-perms/ /docs/fanatical-support-aws/user-management-and-perms/ 301
```

Comment:
removing 301, since it seems to be used by default according to the docs.
```suggestion
/docs/fanatical-support-aws/access-and-permissions/user-management-and-perms/ /docs/fanatical-support-aws/user-management-and-perms/
```